src/sinol_make/executors/time.py

```diff
@@ -16,11 +16,11 @@ def _wrap_command(self, command: List[str], result_file_path: str) -> List[str]:
         if sys.platform == 'darwin':
             time_name = 'gtime'
         elif sys.platform == 'linux':
-            time_name = 'time'
+            time_name = '/usr/bin/time'
```

Comment:
Can we refrain from hardcoding paths to system binaries?
If the only reason for this was that we want to use the GNU time instead of a shell one, then '\time' should suffice.

Reply:
fixed